client/src/app/product/[productid]/page.tsx

```
@@ -22,23 +31,47 @@
+  const [hoveredRating, setHoveredRating] = useState(0);
+  const { user } = useUser();
   console.log(category);
 
   useEffect(() => {
     if (category) {
       fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/category/${category}`)
-        .then(response => response.json())
-        .then(data => {
+        .then((response) => response.json())
+        .then((data) => {
           if (data && data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)